app/src/main/java/com/github/se/cyrcle/ui/report/ParkingReportScreen.kt

```diff
@@ -0,0 +1,188 @@
+                  MaterialTheme.typography.titleMedium.copy(
+                      fontWeight = FontWeight.Bold, textAlign = TextAlign.Start),
+              modifier =
+                  Modifier.fillMaxWidth().padding(vertical = 8.dp).testTag("ReportTitleText"))
+
+          Column(
+              modifier =
+                  Modifier.fillMaxWidth()
+                      .padding(start = 16.dp, bottom = 16.dp)
+                      .testTag("ReportBulletPoints"),
+              verticalArrangement = Arrangement.spacedBy(8.dp)) {
+                BulletPoint(
+                    stringResource(R.string.report_bullet_point_1), testTag = "BulletPoint1")
+                BulletPoint(
+                    stringResource(R.string.report_bullet_point_2_parking),
+                    testTag = "BulletPoint2")
+                BulletPoint(
+                    stringResource(R.string.report_bullet_point_3), testTag = "BulletPoint3")
+              }
+
+          // Select Reason
+          EnumDropDown(
+              options = ParkingReportReason.entries,
+              selectedValue = selectedReason,
+              label = stringResource(R.string.report_reason),
+              modifier = Modifier.testTag("ReasonDropDown"))
+
+          // Additional Details Input
+          ConditionCheckingInputText(
+              value = reportDescription.value,
+              onValueChange = { reportDescription.value = it },
+              label = stringResource(R.string.report_details),
+              minCharacters = 0,
+              maxCharacters = 256,
+              modifier =
+                  Modifier.fillMaxWidth()
+                      .padding(horizontal = horizontalPaddingScaleFactor)
+                      .testTag("ReportDetailsInput"))
+
+          val validInputs = areInputsValid(reportDescription.value)
+
+          if (showDialog) {
+            ReportScreenAlertDialog(
+                onDismiss = { showDialog = false },
+                onAccept = {
+                  showDialog = false
+                  if (validInputs) onSubmit()
+                  navigationActions.goBack()
+                })
+          }
+
+          Button(
+              onClick = { showDialog = true },
+              modifier =
+                  Modifier.testTag("SubmitButton")
+                      .padding(top = 16.dp)
+                      .align(alignment = Alignment.CenterHorizontally),
+              colors = ButtonDefaults.buttonColors(containerColor = Color.Transparent)) {
+                Text(
+                    text = stringResource(R.string.attributes_picker_bottom_bar_submit_button),
+                    color = if (validInputs) MaterialTheme.colorScheme.primary else disabledColor(),
+                    fontWeight = FontWeight.Bold,
+                    style = Typography.headlineMedium,
+                    textAlign = TextAlign.Center)
+              }
+        }
+  }
+}
+
+@Composable
+fun BulletPoint(text: String, testTag: String) {
```

Comment:
I think it's really nice that you created this bullet point composable. It would be perfect if you moved it to the file for Text atoms

Reply:
Agreed ,that makes a very good adding to our set of atoms!